cmd/admin-replicate-rm.go

```diff
@@ -41,7 +41,7 @@ var adminReplicateRemoveFlags = []cli.Flag{
 }
 
 var adminReplicateRemoveCmd = cli.Command{
-	Name:         "remove",
+	Name:         "rm",
```

Comment:
keep and alias for `remove` @poornas for backward compatibility.

Reply:
We could actually hide them @poornas

Reply:
did explore that - looks like only flags can be hidden, not aliases

Reply:
Use the new minio/cli v1.24.1 version and use the `HiddenAliases: true` flag

Reply:
the cli change does not work fully - it hides from the help as expected. However, it doesn't recognize the alias anymore